kim_cole/assignments/birthday_assignment/birthdayCountdown.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<title>Birthday Countdown</title>
+</head>
+Hello Kim Cole!
+<body>
+	<script type="text/javascript">
+
+	function birthdayCountdown(days){
+		var i;
+		for (var i=days; i >0; i--);
+			if (i >=30){
+				console.log(i  + days +' until my birthday. Such a long time... :(');
+			}
+			else 
+			if (i < 30 && i >=5) {
+				console.log(i + days + 'until my birthday. It\'s almost here!');
+			}
+			else 
```

Comment:
the else if should be on the same line, so it should look more like:
if(condition){
    do something
}
else if(condition){
     do something else
}
else {
     do this
}